default.nix

```diff
@@ -50,12 +50,12 @@ in
           sha256 = "13lim8vv78m9lhn7qfjswg7ax825gn0v75gcb80hckxawgk8zxc1";
         };
 
-        chainweb = enableDWARFDebugging (overrideCabal super.chainweb (drv: {
+        chainweb = justStaticExecutables (enableDWARFDebugging (overrideCabal super.chainweb (drv: {
```

Comment:
Can you put a conditional on this so we can leave everything the same in CI?

Reply:
It's CI that needs to be producing the static executable, no?

Reply:
CI should do both.

Reply:
What is the benefit of dynamically linked binaries, if we are not going to deploy those?